### PR TITLE
Populate TPCH tables in Delta Lake query runner

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Populate TPCH tables in Delta Lake query runner

## Documentation

(x) No documentation is needed.

## Release notes

(x) No release notes entries required.

### CODE REVIEW

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java [hunk 1]
```diff
@@ -190,64 +202,100 @@ private static String requiredNonEmptySystemProperty(String propertyName)
...
+
             Thread.sleep(10);
             Logger log = Logger.get(DeltaLakeQueryRunner.class);
             log.info("======== SERVER STARTED ========");
             log.info("\n====\n%s\n====", queryRunner.getCoordinator().getBaseUrl());
-
-            System.out.println("Data directory is: " + queryRunner.getCoordinator().getBaseDataDir().resolve(DELTA_CATALOG));
-            String dataPath = DeltaLakeQueryRunner.class.getClassLoader().getResource("databricks/person").toExternalForm();
-            queryRunner.execute(
-                    format("CREATE TABLE person (name VARCHAR(256), age INTEGER) WITH (location = '%s')", dataPath));
```
Comment (findepi) [inline plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java:217]: is this gone?

**Comment by ebyhr** [comment]
Rebased on upstream to resolve conflicts at https://github.com/trinodb/trino/pull/11717/files#diff-3bc4fc0adce9e4f9a884c37c6a2330eb8f3583fc5932834a0bf0616baaa626efR318

**Review by findepi — COMMENTED** [review]
LGTM

**Comment by findepi** [comment]
still LGTM :)

**Pull request merged**